prompting/organic/organic_scoring_prompting.py

```diff
@@ -226,37 +220,38 @@ async def _wait_for_completion(uid: int):
         return responses
 
     @override
-    async def _query_miners(self, sample: OrganicEntry) -> dict[str, SynapseStreamResult]:
+    async def _query_miners(self, sample: dict[str, Any]) -> dict[str, SynapseStreamResult]:
         """Query miners with the given synthetic or organic sample."""
-        if sample.is_organic and not settings.ORGANIC_REUSE_RESPONSE_DISABLED:
+        if sample.get("organic", False) and not settings.ORGANIC_REUSE_RESPONSE_DISABLED:
             responses = await self._reuse_organic_response(sample)
             return responses
 
         # Get the list of uids to query.
         uids = self._get_random_uids_fn()
         logger.info(f"[Organic] Querying miners with synthetic data, UIDs: {uids}")
-        streams_responses = await dendrite.forward(
-            axons=[settings.METAGRAPH.axons[uid] for uid in uids],
-            synapse=StreamPromptingSynapse(roles=sample.roles, messages=sample.messages),
-            timeout=settings.ORGANIC_TIMEOUT,
-            deserialize=False,
-            streaming=True,
-        )
+
+        async with dendrite(wallet=settings.WALLET) as dend:
+            streams_responses = await dend(
+                axons=[settings.METAGRAPH.axons[uid] for uid in uids],
+                synapse=StreamPromptingSynapse(roles=sample["roles"], messages=sample["messages"]),
+                timeout=settings.ORGANIC_TIMEOUT,
+                deserialize=False,
+                streaming=True,
+            )
```

Comment:
Can you explain to me why you changed this from `await` to `async with`?

Reply:
It works for both cases. Async with just ensures we are using proper asyncio loop, with the given context